types/ramda/test/complement-tests.ts

```diff
@@ -20,9 +20,6 @@ import * as R from "ramda";
     isLengthNotEqual("FOO", "BAR");
     isLengthNotEqual("BAZ", 4); // => true
 
-    // $ExpectType (value: any) => value is any
```

Comment:
Is `// $ExpectType <T>(value: T) => value is Exclude<T, (null | undefined) & T>` an option?

Reply:
I meant to have come back to this one to fix and forgot. Thank you for finding it. What you have is correct and I'll update it